lightly/api/api_workflow_client.py

```diff
@@ -104,10 +104,14 @@ def __init__(
                     )
                 )
         except (
+            # TODO(guarin, 07/23): Why do we have to catch value errors?
             ValueError,
```

Comment:
Do you remember @philippmwirth?

Reply:
`version_compare` raises a `ValueError` if the version is in an unexpected format.

Reply:
Aah, thanks!